src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/FinancialTransactionController.java

```diff
@@ -27,4 +31,13 @@ public ResponseEntity<FinancialTransactionDTO> findTransactionById(@Min(1) @NotN
         return new ResponseEntity<>(financialTransaction, HttpStatus.OK);
 
     }
+
+    @PatchMapping("/{id}")
+    public ResponseEntity<FinancialTransactionDTO> updateTransactionById(
+            @Min(1) @NotNull @PathVariable Long id,
+            @Valid @RequestBody FinancialTransactionDTO updateTransactionDTO) {
+
+        FinancialTransactionDTO financialTransaction = financialTransactionService.updateTransaction(id, updateTransactionDTO);
```

Comment:
Please rename this variable `financialTransaction` to `financialTransactionDTO` or just `dto`

Reply:
Ok. Done